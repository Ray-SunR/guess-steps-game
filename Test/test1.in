gam 	
gama
		


game 1 1 test1A.txt test1B.txt
qui
quii
quiii
quiiitquit
quit
