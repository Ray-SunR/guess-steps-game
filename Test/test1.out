Total is 1, max deduction is 1
Player A's move
Total is 0
Player A wins
Score is
A 1
B 0
